content/docs/operating/security.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Security
+sort_rank: 4
+---
+
+# Security Model
+
+Prometheus is sophisticated system with many components and many integrations with
+other systems. It can be deployed in a varietry of trusted and untrusted
+enviroments.
+
+This page describes the general security assumptions of Prometheus and the
+attack vectors that some configurations may enable.
+
+As with any complex systems it is not possible to gaurantee that there are no
+bugs. If you find a security bug, please file it in the issue tracker of the
+relevant component.
```

Comment:
We've had by my count 2 security bugs (one XSS, one secret exposure), I'd currently prefer to risk a bad bug being public early than pointing more support load directly at -team.